unit_tests/recovery/ut_database.cpp

```diff
@@ -18,14 +18,17 @@
 
 #include "ut_database.hpp"
 
+#include <limits>
+
 /*******************************************************
  * ut_database_values_generator
  *******************************************************/
 
 ut_database_values_generator::ut_database_values_generator (const ut_database_config &a_database_config)
   : m_database_config (a_database_config)
-  , m_lsa_log_id (10100) // just starts from an arbitrary value
   , m_gen (m_rd ())
+  //, m_log_lsa () // the log_lsa ctor actually does not set any value
```

Comment:
About log_lsa ctor not actually setting any value, what do you think of making it set?

```
struct log_lsa
{
  std::int64_t pageid:48 = 0;		/* Log page identifier : 6 bytes length */
  std::int64_t offset:16 = 0;		/* Offset in page : 2 bytes length.
                                              offset == 'area offset' */
```

Reply:
Nothing against. I was thinking conservatively to not alter a class whose behavior is key in many contexts.